cloud_experts_tutorials/cloud-experts-using-alb-and-waf.adoc

```diff
@@ -21,24 +21,21 @@ AWS WAF is a web application firewall that lets you monitor the HTTP and HTTPS r
 
 You can use an AWS Application Load Balancer (ALB) to add a Web Application Firewall (WAF) to your {product-title} (ROSA) workloads. Using an external solution protects ROSA resources from experiencing denial of service due to handling the WAF.
 
-[NOTE]
+[IMPORTANT]
 ====
-It is recommended that you use the xref:../cloud_experts_tutorials/cloud-experts-using-cloudfront-and-waf.adoc#cloud-experts-using-cloudfront-and-waf[CloudFront method] unless you absolutely must use an ALB based solution.
+It is recommended that you use the significantly more flexible xref:../cloud_experts_tutorials/cloud-experts-using-cloudfront-and-waf.adoc#cloud-experts-using-cloudfront-and-waf[CloudFront method] unless you absolutely must use an ALB based solution.
 ====
 
-//[Here](https://iamondemand.com/blog/elb-vs-alb-vs-nlb-choosing-the-best-aws-load-balancer-for-your-needs/)'s a good overview of AWS LB types and what they support
-
-// Loosely based off EKS instructions here - https://aws.amazon.com/premiumsupport/knowledge-center/eks-alb-ingress-aws-waf/
-
 [id="prerequisites_{context}"]
 == Prerequisites
 
+* A multi-AZ ROSA (HCP or Classic) cluster.
```

Comment:
```suggestion
* A multiple availability zone (AZ) ROSA or ROSA (classic architecture) cluster.
```

Reply:
I know we're going to be changing the naming, but it hasn't happened across any of the other ones that I published today, so I kept the (HCP or Classic) until we can make all of these changes across the docs.